v1/examples/pipeline/steps/custom-approval-step.yaml

```diff
@@ -0,0 +1,26 @@
+version: 1
+kind: pipeline
+spec:
+  stages: 
+    - type: custom
+      name: custom
+      id: custom
+      spec:
+        steps:
+          - type: custom-approval
+            spec:
+              approval_criteria: 
```

Comment:
Can we shorten this? Just "criteria", for example.

Reply:
We have a rejection_criteria as well, hence we cannot name it to criteria